### DISCUSSION TITLE
Markdown / LaTeX

### DISCUSSION THREAD

**Discussion by tristanbouchard** [codepuzzle-io/www.codepuzzle.io#13] [Idées / Ideas]
Bonjour,

Serait-il possible d'intégrer du code LaTeX comme le reste du code Markdown ?

Merci

**Comment by laurentabbal** [comment]
C'est noté. Je fais ça dans une dizaine de jours. Ne pas hésiter à relancer si j'oublie.

**Comment by tristanbouchard** [comment]
Bonjour,

Merci beaucoup pour votre réactivité sur l'autre message d'erreur et sur celui-ci.
Si vous avez besoin d'aide, je suis prêt à aider à ma maigre hauteur.
